### PR TITLE
Guide and API pages for version 0.19.0

### PR DESCRIPTION
**Pull request by shivag46**
## PR Checklist

- [ ] I have read and signed the [Contributor License Agreement](https://forms.gle/PZyiY5N2GDQU8GsV9).

### CODE REVIEW

File: new-site/docs-site/content/0.19.0/guide/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+If you had installed Typesense from a DEB/RPM package, the Typesense server is automatically started as a systemd service when installation is complete. You can check the status via:
+
+`systemctl status typesense-server`
+
+By default, Typesense will start on port 8108, and the installation will generate a random API key, which you can view/change from the configuration file at `/etc/typesense/typesense-server.ini`
+
+## From the pre-built binary
+If you have downloaded the pre-built binary, you can start Typesense with minimal options like this:
+
+    mkdir /tmp/typesense-data
```
Comment (jasonbosco) [inline new-site/docs-site/content/0.19.0/guide/README.md:49]: Could you surround this code block and other similar by 

```shell
```shell
...
```

So it's syntax highlighted?

File: new-site/docs-site/content/0.19.0/guide/installClient.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Installing a client
+At the moment, we have clients for Javascript, PHP, Python, Ruby.
```
Comment (jasonbosco) [inline new-site/docs-site/content/0.19.0/guide/installClient.md:2]: Could you link each of these to the git repo for each library?

File: new-site/docs-site/content/0.19.0/api/usage/create.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
+# Create a collection
```
Comment (jasonbosco) [inline new-site/docs-site/content/0.19.0/api/usage/create.md:1]: Could you nest all collection related operations under a single "Collections" section and then all document related operations under a single "Documents" section? 

So something like:

- Collections
  - Create a collection
  - Retrieve a collection
  - List all collections
  - Drop a collection
  - Etc
- Documents 
  - Index a document (this is missing btw)
  - Search for documents
  - Import documents
  - Etc

**Review by jasonbosco — CHANGES_REQUESTED** [review]

File: new-site/docs-site/content/0.19.0/api/usage/create.md (hunk 3 above)
Comment (shivag46) [reply new-site/docs-site/content/0.19.0/api/usage/create.md:1]: Fine. Will work on that

**Review by jasonbosco — CHANGES_REQUESTED** [review]
@shivag46 The structure looks good now. 

I noticed that the sidebar requires elaborate configuration to be set in `config.js`. So I'm going to write a quick module to generate the sidebar programmatically, by reading the directory structure and using the file names as the side bar entries. In preparation for this, could you rename all the files and directories in `0.19.0` to match the headings you've defined in `sidebar` in `config.js`? Could you also use `snake-case` for multi-word file names?

Separately, could you double-check to make sure that all content for v0.19 has been moved over to the new-site? 

We're going to essentially be using v0.19 docs to quickly generate docs for the older versions. So any content issues we let slip now is going to get very time-consuming to make changes later across all versions later, once we've created the older docs versions...

**Comment by jasonbosco** [comment]
Actually, looks like generating the sidebar configs programatically is a little more complicated than I thought, considering the sort order of the entries on the sidebar. We'd have to use a sort key in the filename, which makes it even more complicated to manage. So let's just stick with what you already have for the sidebar config.

But could you still rename the files to make them use the snake-cased version of the section title you have in the sidebar config? So for eg, given this side bar config: `['/0.19.0/api/cluster','Cluster operations']` you want to name rename the file from `cluster.md` to `cluster-operations.md`.

**Comment by shivag46** [comment]
Sure!! I will do that

**Review by jasonbosco — APPROVED** [review]

**Pull request merged**